README.md

```diff
@@ -43,10 +43,16 @@ $ eval $(docker-machine env mybox)
 
 Also, you should add a route to access containers inside your VM.
 
+For Linux
 ```
 $ sudo route -n add 172.0.0.0/8 $(docker-machine ip $(docker-machine active))
 ```
 
+For Windows ( requires admin elevation )
```

Comment:
You should create a new section like for `Linux` or `Mac OS` or update the Mac OS section

Reply:
ok ... sorry